projects/ad74413r/src/examples/dummy/dummy_example.c

```diff
@@ -49,7 +48,7 @@
  * @return ret - Result of the example execution. If working correctly, will
  *               execute continuously the while(1) loop and will not return.
 *******************************************************************************/
-int dummy_example_main()
+int example_main()
```

Comment:
using a common name like `example_main()` allows us to delete the various headers `dummy_example.h` which are totally unnecessary